app/src/main/kotlin/lv/rigadevday/android/ui/tabs/TabActivity.kt

```diff
@@ -21,8 +22,10 @@ class TabActivity : BaseActivity() {
 
     private val scheduleFragment: Fragment by lazy { MyScheduleFragment() }
     private val speakersFragment: Fragment by lazy { SpeakerListFragment() }
-    private val venuesFragment: Fragment by lazy { PlaceholderFragment.newInstance("venues") }
     private val partnersFragment: Fragment by lazy { PartnersFragment() }
+    private val venuesFragment: Fragment by lazy { VenuesRootFragment() }
+    private val organizersFragment: Fragment by lazy { PlaceholderFragment.newInstance("organizers") }
```

Comment:
Rebase accident I assume. Organizers/Partners fragment is already there. Please check this again.